tests/kernel/wave/wave_gemm_test.py

```diff
@@ -161,6 +167,19 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
             dump_perf, "tk_" + perf_filename
         )
 
+    dynamic_symbols = []
+    dynamic_symbols_map = {}
+    if dynamic_dims:
+        dynamic_symbols_map[M] = hyperparams[M]
+        dynamic_symbols_map[N] = hyperparams[N]
+        dynamic_symbols_map[K] = hyperparams[K]
+        dynamic_symbols.append(M)
+        dynamic_symbols.append(N)
+        dynamic_symbols.append(K)
```

Comment:
maybe we should move this to the hyper param?

Reply:
actually nvm, hyperparams require the key to be symbols, doesn't have to be in this PR but would be nice to think of some way to clean this up, i.e make this a bit more ergonomic for kernel/torch model authors.

Reply:
This is only required for the pytest. For kernel authors, they would not specify the dimensions in the hyper parameter dictionary.